### PR TITLE
Split 06-collab in two different topics

### PR DESCRIPTION
**Pull request by iglpdc**
This is a fix to #113, if we finally decide to do it.

I've chosen "Remotes In Github" as the name for the new topic. Other suggestions are welcome.

The rest is just renaming the files, and moving things around.

### CODE REVIEW

**Comment by iglpdc** [comment]
This is ready to review. Can someone take a look?

**Comment by msarahan** [comment]
+1, good split.

**Comment by PBarmby** [comment]
Agreed, split seems sensible, since remotes are useful even when your collaborator is yourself.

**Comment by iglpdc** [comment]
Thanks @PBarmby and @msarahan. I'm merging.

File: 08-collab.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+For the next step, get into pairs.
+Pick one of your repositories on GitHub to use for collaboration.
+
+> ## Practicing by yourself {.callout}
+>
+> If you're working through this lesson on your own, you can carry on by opening
+> a second terminal window, and switching to another directory (e.g. `/tmp`).
+> This window will represent your partner, working on another computer. You
+> won't need to give anyone access on GitHub, because both 'partners' are you.
```
Comment (swaldman3) [inline 08-collab.md]: suggest "both partners are you, using your Github login" to make it clearer what is meant here.

Comment (iglpdc) [inline 08-collab.md]: Ok, @swaldman3, I agree in doing the change in the callout. This PR is merged already, so I'll have to make a separate one from scratch. Do you wanna do the PR yourself?

Comment (swaldman3) [inline 08-collab.md]: Happy to, but it won't be today. If you're wanting to get it wrapped up, please go ahead and do it :-)

Comment (iglpdc) [inline 08-collab.md]: El 11/05/2015, a las 10:51, Simon Waldman <notifications@github.com> escribió:
> Happy to, but it won't be today. If you're wanting to get it wrapped up, please go ahead and do it :-)
> 
> 
Good, I'll leave to you then, so you get credit for it.

**Pull request merged**